attack.go

```diff
@@ -186,7 +186,7 @@ func (h headers) Set(value string) error {
 	if key == "" || val == "" {
 		return fmt.Errorf("header '%s' has a wrong format", value)
 	}
-	h.Add(key, val)
+	h.Header[key] = []string{val}
```

Comment:
The method is called `Set` in order to implement the `flag.Value` interface: http://golang.org/pkg/flag/#Value
Yes, it's used to set an arbitrary number of headers that have the same key which are valid HTTP semantics.